src/hand-tracking/CMakeLists.txt

```diff
@@ -153,6 +153,7 @@ set(${EXE_TARGET_NAME}_HDR
         include/VisualSIS.h
 		include/WalkmanArmModel.h
 		include/WalkmanCamera.h
+        include/utils.h
```

Comment:
Wrong indentation.

Reply:
Done in a separate commit since there were many indentation issues!